plugin/src/utils/cache.ts

```diff
@@ -4,7 +4,7 @@ import fsStore from "cache-manager-fs-hash"
 import path from "path"
 import rimraf from "rimraf"
 
-import store from "~/store"
+import { default as pluginStore } from "~/store"
```

Comment:
i did this to prevent collisions below with `store` module and `store` as an argument, but that seems to be causing errors now i think?

Reply:
huh 🤔 maybe change it back? It's definitely working as-is. I think there wont be a collision since the local scope will override the module scope.

Reply:
hmm... that did not fix the issue, ill keep looking!

Reply:
i know re-using variable names in this case doesn't break things because of scope, but it *does* drive me crazy, on principle 😆

Reply:
😆 fair enough, haha. I don't like it much either. It can make things pretty confusing! This code was copy/pasted from core to add a hard caching layer that works similar to the core caching layer. So I didn't change very much code in it ;)

Reply:
ok! keeping it as is, in terms of runtime impacting code, then.